crates/core/src/db/datastore/traits.rs

```diff
@@ -173,12 +172,8 @@ pub struct TxRecord {
     pub(crate) op: TxOp,
```

Comment:
Do we need `TxOp::Insert` still?

It does not seem like `DataKey` is used anywhere, so `TxData` could possibly replaced by `spacetimedb_commitlog::TxData`.